app.rb

```diff
@@ -18,7 +20,7 @@ def run
 
```

Comment:
![image](https://github.com/sergio-usma/ruby_school_library/assets/93039359/c7b05cc5-a134-4ed2-9d53-1ca79edb0609)

- [ ] It would be nice if you fix the highlighted link in the Table of Contents which you can see in the image above because right now it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍